AntennaTracker/GCS_Mavlink.cpp

```diff
@@ -234,42 +229,9 @@ bool GCS_MAVLINK_Tracker::try_send_message(enum ap_message id)
         CHECK_PAYLOAD_SIZE(HWSTATUS);
         tracker.send_hwstatus(chan);
```

Comment:
:-)

I agree this should work just like the others.  This way I don't need to work out why it was done this way!

Happy to remove it as another commit if @rmackay9 says it can e like all of the others.